internal/configuration/settings/serverselection.go

```diff
@@ -107,6 +123,11 @@ func (ss *ServerSelection) validate(vpnServiceProvider string,
 		return fmt.Errorf("for VPN service provider %s: %w", vpnServiceProvider, err)
 	}
 
+	err = validateFeatureFilters(*ss, vpnServiceProvider)
+	if err != nil {
+		return fmt.Errorf("for VPN service provider %s: %w", vpnServiceProvider, err)
+	}
+
 	if *ss.OwnedOnly &&
 		vpnServiceProvider != providers.Mullvad {
```

Comment:
Why not incorporate OwnedOnly etc. (below) within validateFeatureFilters?

Reply:
Not sure if this was a feature or a "subscription tier" filter. If it is a feature then it makes sense to move it as well!

Reply:
Good point indeed, I've separated the validation in `validateSubscriptionTierFilters` and `validateFeatureFilters`